wordpress-mobile/WordPress-iOS
# History / Revision: Loading a selected revision

Refs. #10304 

This PR add the possibility to load a post form a revision and update the current post content.

`AbstractPost` contained already part fo the logic to map a local created revision and assign the origin post. I added a new method which update the current post content, title and modified date from a given revision post.

`RevisionsTableViewController` and `RevisionBrowserState ` are conformance to the `SelectedRevisionLoadedProtocol`  protocol, which contain a closure as property. This escaping closure will be triggered as soon the post local service will load the revision post, and it will update the current post.

**To Test**:
- Select a post with a history list
- Select a revision and open the Revisions browser
- Click on _Load_. The call should succeed and you should see the post title and content changed.
